x-pack/plugins/infra/public/components/asset_details/content/content.tsx

```diff
@@ -22,7 +30,7 @@ export const Content = () => {
             ContentTabIds.LOGS,
             ContentTabIds.METADATA,
             ContentTabIds.PROCESSES,
-            ContentTabIds.PROFILING,
+            ...(loading || showProfilingEmptyState ? [] : [ContentTabIds.PROFILING]),
```

Comment:
I think it might be better to hide this global date picker from profiling all the time and just use a separate `<DatePicker>` within the profiling tab. This way profiling state won't leak outside of the tab. Wdyt?